### PR TITLE
Added the code for finding all Articulation Points in given graph

### PR DESCRIPTION
**Pull request by devartstar**

### CODE REVIEW

**Comment by yashrsharma44** [comment]
Hi @devartstar can you add in an example and an output for cross-checking?

**Comment by devartstar** [comment]
Example Case 1 -
Screenshot from 2021-01-08 12-31-41

**Comment by devartstar** [comment]
Example Case 2 ---
Screenshot from 2021-01-08 12-41-24

File: articulationPoint.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+const GraphBase = require('./graph-base');
+const Queue = require('./ds/queue');
+
+class ArticulationPoint extends GraphBase {
+    constructor(numVertices = 0, numEdges = 0, startingPoint = 0) {
+        super(numVertices, numEdges);
+    }
+
+    findArtiPt(graph, v, time) {
```
Comment (yashrsharma44) [inline articulationPoint.js:9]:
```suggestion
    findArticulationPoint(graph, v, time) {
```

File: articulationPoint.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        // console.log("hello from node "+v);
+        graph[v].visited = true;
+        time++;
+        graph[v].disc = time;
+        graph[v].low = time;
+        var child = 0;
+        // console.log(graph[v].al.length);
```
Comment (yashrsharma44) [inline articulationPoint.js:16]: Could you remove unused console.log statements?

**Review by yashrsharma44 — CHANGES_REQUESTED** [review]
Sounds great! Make the suggested changes, and the PR is good to go!

File: articulationPoint.js (hunk 2 above)
Comment (devartstar) [reply articulationPoint.js:16]: okay

File: articulationPoint.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
+const GraphBase = require('./graph-base');
+const Queue = require('./ds/queue');
+
+class ArticulationPoint extends GraphBase {
+    constructor(numVertices = 0, numEdges = 0, startingPoint = 0) {
+        super(numVertices, numEdges);
+    }
+
+    findArtiPt(graph, v, time) {
+        // console.log("hello from node "+v);
```
Comment (yashrsharma44) [inline articulationPoint.js:10]: Can you remove this as well?
```suggestion
```

**Review by yashrsharma44 — CHANGES_REQUESTED** [review]

**Comment by yashrsharma44** [comment]
Merging the PR, thanks for the changes! @devartstar 

Github Tip - After addressing the suggestions please click the resolve button, which helps the reviewer check which suggestions have been addressed!

**Pull request merged**